package.json

```diff
@@ -16,10 +16,8 @@
     "build-system-uri": "cd system_uri && cargo build && cd ../ && cp -f system_uri/target/debug/*system_uri* ./src/native",
     "build-safe-lib": "cd native/safe_app && cargo build --features \"use-mock-routing testing\" && cd ../../ && cp -f native/target/debug/*safe_app* ./src/native",
     "build-libs": "npm run build-system-uri && npm run build-safe-lib",
```

Comment:
Yes. Though we might want to reconsider the submodule-part now. It is a pain to download the entire repo on clone ..